coralproject/talk
# Add user info to moderation queue

## What does this PR do?

Adds user info to the moderation queue. Adds actions to the moderation queue endpoint for future use.

## How do I test this PR?

1. Create a pending comment.
2. Visit `http://localhost:3000/admin`
3. Pending comments should list author name.
